src/pages/Payment.vue

```diff
@@ -132,6 +132,9 @@ export default {
             this.$router.push("/account");
         },
         onRequestPayment() {
+            document
+                .getElementById("requestPayment")
+                .setAttribute("disabled", "disabled");
```

Comment:
check this doesn't remain disabled. after an error [assumption, they might want to try again]; we might do this by sticking a line in the second function that pendRecVouchers takes, which is it's "failure" function.

Reply:
Done.